ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -100,8 +97,6 @@
       .build();
 
   private final AtomicBoolean acceptingStatements = new AtomicBoolean(true);
-
-  private final Map<QueryId, PersistentQueryMetadata> persistentQueries;
```

Comment:
To support the sandbox being able to track persistent queries the state the tracks them has been moved into the `EngineContext`.